src/TextField/TextFieldLabel.js

```diff
@@ -4,15 +4,14 @@ import React, { PropTypes } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import classNames from 'classnames';
 import { easing } from '../styles/transitions';
+import LabelBase from '../internal/LabelBase';
```

Comment:
Oh, I see what you mean. The material specification doesn't say anything about it. We are quite on our own.
That sounds good to me to display some errors at the selection level. How do you think it should looks like?